lrauv_ignition_plugins/src/ScienceSensorsSystem.cc

```diff
@@ -142,22 +150,57 @@ class tethys::ScienceSensorsSystemPrivate
   /// \brief Octree for data search based on spatial location of sensor. One
   /// octree per point cloud in timeSpaceCoords.
   public: std::vector<pcl::octree::OctreePointCloudSearch<pcl::PointXYZ>>
-            spatialOctrees;
-
-  /// \brief Set to true after the spherical coordinates have been initialized.
-  /// This may happen at startup if the SDF file has them hardcoded, or at
-  /// runtime when the first vehicle is spawned. Assume the coordinates are
-  /// only shifted once.
-  public: bool sphericalCoordinatesInitialized{false};
```

Comment:
This is reverting #77, which was already on `main`. But it's not functional, so I think it's ok to remove it for now and revisit at #70.

Reply:
It's being restored in #98